packages/mdc-list/README.md

```diff
@@ -39,7 +39,7 @@ npm install @material/list
 ### HTML Structure
 
 ```html
-<ul class="mdc-list" aria-orientation="vertical">
```

Comment:
Why was aria-orientation removed from examples? The default is documented as horizontal.

Reply:
`listbox` is applicable only if one of the list item is selectable. But in our baseline version we don't add that support. 

I'll add a note about aria orientation in accessibility section.

Reply:
`role="list"` or `ul/li` elements are supposed to be keyboard interactive as per ARIA spec. I'm inclined towards using `aria-orientation` only when applicable. User may not initialize MDCList on non-interactive lists. WDYT?

Reply:
`aria-orienation` is used to indicate which arrow keys the foundation listens to for traversing the list. If we remove this from all `ul` examples that are not `role="listbox"` we need to change the attribute we use to indicate list direction.

Reply:
Response to your comment through email: 

I understand that it's not a part of the spec, but this functionality is specifically what we wanted to make lists more accessible regardless if you can make a selection or not.

Reply:
I don't think the spec specifically disallows this behavior, it's just not mentioned IIRC.